[2023-09-21 16:10:55,392] [query_look_up_table_new_fix_first_response.py:create_query_lookup_table] : response number: 0
